### PR TITLE
Fix path

### PR DESCRIPTION
**Pull request by nbm-se**

### CODE REVIEW

**Review by alexandrorlenko — CHANGES_REQUESTED** [review]
Please provide a description

File: INSTALL.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ For installing Go lang on Linux [see.](https://golang.org/doc/install)
 ## Clone repo
 
 ```git clone git@github.com:setronica-dev/product-catalog-import-tool.git```
-```cd /ts```
+```cd ./product-catalog-import-tool```
```
Comment (alexandrorlenko) [inline INSTALL.md:15]: could you please change it in a way to show them in two lines instead of one?

**Review by alexandrorlenko — CHANGES_REQUESTED** [review]

File: INSTALL.md (hunk 1 above)
Comment (nbm-se) [reply INSTALL.md:15]: Yes, of course

**Review by alexandrorlenko — APPROVED** [review]
it still requires a description and commits squashing but the code is fine

**Pull request merged**